doc/install/easy-install.md

```diff
@@ -47,27 +66,16 @@ conda create -n deepmd deepmd-kit=*=*gpu libdeepmd=*=*gpu lammps cudatoolkit=11.
 ```
 One could change the CUDA Toolkit version from `10.2` or `11.6`.
 
-One may specify the DeePMD-kit version such as `2.1.1` using
+One may specify the DeePMD-kit version such as `2.2.9` using
 ```bash
-conda create -n deepmd deepmd-kit=2.1.1=*cpu libdeepmd=2.1.1=*cpu lammps horovod -c https://conda.deepmodeling.com -c defaults
+conda create -n deepmd deepmd-kit=2.2.9=*cpu libdeepmd=2.2.9=*cpu lammps horovod -c https://conda.deepmodeling.com -c defaults
 ```
 
 One may enable the environment using
 ```bash
 conda activate deepmd
 ```
 
-### conda-forge channel
-
-DeePMD-kit is also available on the [conda-forge](https://conda-forge.org/) channel:
-
-```bash
-conda create -n deepmd deepmd-kit lammps horovod -c conda-forge
-```
-
-The supported platform includes Linux x86-64, macOS x86-64, and macOS arm64.
-Read [conda-forge FAQ](https://conda-forge.org/docs/user/tipsandtricks.html#installing-cuda-enabled-packages-like-tensorflow-and-pytorch) to learn how to install CUDA-enabled packages.
-
 ## Install with docker
 A docker for installing the DeePMD-kit is available [here](https://github.com/orgs/deepmodeling/packages/container/package/deepmd-kit).
```

Comment:
Update dead link
```suggestion
A docker for installing the DeePMD-kit is available [here](https://github.com/deepmodeling/deepmd-kit/pkgs/container/deepmd-kit).
```

Reply:
It seems not dead but redirects to the new link.